# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/tests/mochitest/treeupdate/a11y.ini
# Commit: 89963083c79d
# Full Hash: 89963083c79dd5f7fbb1bcf90efe5f9985597db5
# Author: Eitan Isaacson <eitan@monotonous.org>
# Date: 2019-08-29 21:45:25
# Regressor Bug: 1576690
# File Overlap Count: 1
# Description:
#   Bug 1576690 - Prune de-slotted accessibles, or relocate them to new slot. r=Jamie,emilio
#   
#   This patch does several things:
#    1. If there is a change to a host or a slot, check the slottable
#       elements to see if they are rendered in the tree. Remove them if not.
# ==============================================================================

diff -r e379881b3989 -r 89963083c79d accessible/tests/mochitest/treeupdate/a11y.ini
--- a/accessible/tests/mochitest/treeupdate/a11y.ini	Thu Aug 29 16:02:40 2019 +0000
+++ b/accessible/tests/mochitest/treeupdate/a11y.ini	Thu Aug 29 16:14:48 2019 +0000
@@ -35,6 +35,7 @@
 [test_optgroup.html]
 [test_recreation.html]
 [test_select.html]
+[test_shadow_slots.html]
 [test_shutdown.xul]
 [test_table.html]
 [test_textleaf.html]